src/recursion.erl

```diff
@@ -0,0 +1,31 @@
+-module(recursion).
+
```

Comment:
Yeah, good call. I semi-deliberately left out the exports since I was mainly intending for this code to be a concise, illustrative example, rather than something anyone might ever actually want to run. I suppose it'd be easy enough to just add the one export line though and have a working module, so I'll go ahead and do that.